.travis.yml

```diff
@@ -33,21 +27,62 @@ branches:
     - /^amp-release-.*$/
     - /^revert-.*$/
 addons:
-  chrome: stable
   hosts:
     - ads.localhost
     - iframe.localhost
     # Requested by some tests because they need a valid font host,
     # but should not resolve in tests.
     - fonts.googleapis.com
-  apt:
-    packages:
-      - protobuf-compiler
-      - python-protobuf
-matrix:
+stages:
+  - build
+  - test
+jobs:
   include:
-    - env: BUILD_SHARD="unit_tests"
-    - env: BUILD_SHARD="integration_tests"
+    - stage: build
+      name: "Build"
+      script:
+        - node build-system/pr-check/build.js
+    - stage: build
+      name: "Checks" 
+      script:        
+        - node build-system/pr-check/checks.js
+    - stage: build
+      name: "Validator Tests"        
+      before_script:
+        - pip install --user protobuf
+      script:        
+        - node build-system/pr-check/validator-tests.js
+      addons:
+        apt:
+          packages:
+            - protobuf-compiler
+            - python-protobuf
+    - stage: test
+      name: "Dist, Bundle Size, Single Pass Tests"
+      addons:
+        chrome: stable
+      script:
+        - node build-system/pr-check/dist-tests.js
+    - stage: test
+      name: "Visual Diff Tests"
+      script:
+        - node build-system/pr-check/visual-diff-tests.js
+    - stage: test
+      name: "Local Tests"
+      addons:
+        chrome: stable
+      script:
+        - node build-system/pr-check/local-tests.js
+    - stage: test
+      name: "Remote (Sauce Labs) Tests"
+      script:
+        - node build-system/pr-check/remote-tests.js
+      after_script:
+        - build-system/sauce_connect/stop_sauce_connect.sh
+  fast_finish: true
+  allowed_failures:
```

Comment:
According to https://docs.travis-ci.com/user/customizing-the-build/#rows-that-are-allowed-to-fail, this is `allow_failures`, not `allowed_failures`

For tracking, add a TODO above this section to eventually remove it.

Finally, test this behavior with a follow up dummy PR by injecting a known failure into a test that will run on Sauce Labs.

Reply:
This fixed it, thanks. Example here - https://travis-ci.org/ampproject/amphtml/builds/493911037